python/ray/rllib/ppo/filter.py

```diff
@@ -47,6 +60,10 @@ def update(self, other):
         self._M = M
         self._S = S
 
+    def __repr__(self):
+        return '(n={}, mean={}, std={})'.format(
+            self.n, np.mean(self.mean), np.mean(self.std))
```

Comment:
This shouldn't compute np.mean again, the mean is already the mean of the observation (could be a vector); alternatively we can get rid of the __repr__ function, but as it is now it seems confusing.

Reply:
I re-labelled this to say `mean_mean`. I think it's useful for debugging to have a nice repr, but the full mean is just too long.